mediawiki/stack.yml

```diff
@@ -0,0 +1,21 @@
+# MediaWiki with MariaDB
+#
+# Access via "http://localhost:8080"
+#   (or "http://$(docker-machine ip):8080" if using docker-machine)
+version: '3'
+services:
+  mediawiki:
+    image: mediawiki
+    ports:
+      - 8080:80
+    links:
+      - database
+    volumes:
+      - /var/www/html/images
```

Comment:
@yosifkit Sadly, the web install does not write the `LocalSettings.php` file. I created [T172555](https://phabricator.wikimedia.org/T172555) to help deal with that either by allowing for a default `LocalSettings.php` file that we would put in the container (that would read the environment variables) or by allowing the `LocalSettings.php` to not exist at all and _only_ using environment variables (unless overridden).

The `cli` installer _will_ create the`LocalSettings.php` file, but you would have to be in the container (and have a database service up and running).

I will add your suggestion for now and hopefully in the future we can remove it! :)